test/scripts/basic-test.js

```diff
@@ -203,16 +203,16 @@ describe('<ll-property-images>', function() {
 
   });
 
-  it('should, when currentDefaultImage is undefined, check whether currentDefaultImage gets set to newDefaultImage after calling `_currentDefaultImage()`', function() {
+  it('should (1) check whether currentDefaultImage gets set to newDefaultImage, (2) that newDefaultImage.isDefault is switched  to true, and (3) that oldDefaultImage.isDefault is switched  to false', function() {
```

Comment:
So I was going to add a comment to this while it was still open, but if you are having to put 1,2,3 for a test then it needs to be broken out into 3 tests. At that point your test is trying to assert too many things.